### DISCUSSION TITLE
“View code” is broken on mobile safari

### DISCUSSION THREAD

**Discussion by ibash** [community/community#9670] [Mobile] [answered]
Steps:
1. Open any repo on mobile safari
2. Tap “view code”

Expected: list of files are shown

Actual: button is unresponsive

**Comment by brianlovin** [comment]
Hey @ibash is this still happening for you? I'm unable to reproduce this on my end.

**Comment by ibash** [comment]
Yep still happening - what debugging information can I give you? If you’re in sf and willing to meet can literally give you my phone.

**Reply by brianlovin** [reply]
The most helpful thing would be to share a few more details about your OS version, the repo you're viewing that doesn't work (if public), and check if it's affecting _all_ repositories or just a few. Thank you!

**Comment by ibash** [comment]
1. iPhone 13 mini
2. iOS 15.1.1
3. Mobile safari
3. It's all repos
4. Not affected by being logged in / logged out, reproduces in a private tab

Going to dump picture of all my settings in the next comment

**Reply by brianlovin** [reply]
Okay. Could you make a screen recording here? Also, do you have any iOS Safari extensions that may be interfering here? To check, tap the puzzle icon in the url field and see if anything is enabled here that might be messing with JavaScript or content blocking.

**Reply by ibash** [reply]
I have 1password extension, I tried disabling it but didn't work. Screen recording incoming!

**Reply by ibash** [reply]
https://user-images.githubusercontent.com/204566/151047267-66332a25-37cc-4617-a969-2da5d21a8d74.mp4

**Reply by ibash** [reply]
Oh, I also use nextdns on mobile to avoid ads, but even disabling that doesn't fix the issue.

**Comment by ibash** [comment]
![IMG_0680](https://user-images.githubusercontent.com/204566/151046476-0d5feea5-feae-4a2a-8ff6-c3301013e65c.PNG)
![IMG_0681](https://user-images.githubusercontent.com/204566/151046485-aac2cdc3-8aa9-4d1d-964d-c2d611ca2fdf.PNG)
![IMG_0682](https://user-images.githubusercontent.com/204566/151046490-4fc709c7-7cb8-4f40-aa6b-318ef49eac1f.PNG)
![IMG_0683](https://user-images.githubusercontent.com/204566/151046494-81cb10ea-4cab-4d9d-94c2-5a7c7dc5c04b.PNG)
![IMG_0684](https://user-images.githubusercontent.com/204566/151046499-db4d6525-50c9-402f-8a68-36c835b2712a.PNG)
![IMG_0685](https://user-images.githubusercontent.com/204566/151046500-b57dacec-b2d6-4ab9-9f99-6c036119e5d4.PNG)
![IMG_0686](https://user-images.githubusercontent.com/204566/151046501-6ee866e0-7104-4df9-afa3-1647da46780f.PNG)
![IMG_0687](https://user-images.githubusercontent.com/204566/151046504-d4b037f4-4543-476b-a4e1-912357e9b918.PNG)
![IMG_0688](https://user-images.githubusercontent.com/204566/151046506-d432e542-98d0-496d-bb5a-c59d78d342c6.PNG)
![IMG_0689](https://user-images.githubusercontent.com/204566/151046512-2ccc5e8d-d990-4bd3-a7a4-814640827628.PNG)
![IMG_0690](https://user-images.githubusercontent.com/204566/151046517-6ad2cd40-7e6a-4568-b28c-fb24bd554392.PNG)
![IMG_0691](https://user-images.githubusercontent.com/204566/151046519-c6d76d24-c195-4f6d-9266-40925837ca0f.PNG)
![IMG_0692](https://user-images.githubusercontent.com/204566/151046521-402fcca5-bad8-4454-9582-dc43fe918ab4.PNG)
![IMG_0693](https://user-images.githubusercontent.com/204566/151046522-c2725df7-5530-4aac-a436-218c360fa063.PNG)
![IMG_0694](https://user-images.githubusercontent.com/204566/151046525-6ad92f62-bc12-4dff-9d72-77b141b876d7.PNG)
![IMG_0695](https://user-images.githubusercontent.com/204566/151046527-f37dab26-c69a-416a-b241-8c25f8a1efaa.PNG)
![IMG_0696](https://user-images.githubusercontent.com/204566/151046530-46739ae8-771e-41ad-bfdb-424a6e9a2b6d.PNG)
![IMG_0697](https://user-images.githubusercontent.com/204566/151046531-16674120-4aa1-4b94-a5a0-838c885cfaf5.PNG)
![IMG_0698](https://user-images.githubusercontent.com/204566/151046533-9475e0de-4eab-4cfb-bd49-9f148b863174.PNG)

**Comment by ibash** [comment, marked as answer]
@brianlovin I got this working -- I checked my "experimental features" against an iOS simulator and found a few of them were disabled, which is odd because I don't recall ever changing these. After making them match and restarting it's all working again.

**Reply by brianlovin** [reply]
Thanks for the follow up, very glad it's working now!
